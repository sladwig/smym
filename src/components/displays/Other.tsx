import React from 'react';

export const WhiteDisplay = () => <span>&nbsp;</span>;
export const NullDisplay = () => null;
