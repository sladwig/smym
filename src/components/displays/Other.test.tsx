import { render } from '@testing-library/react';
import React from 'react';
import { Other } from './Other';

describe('Other', () => {
    test('renders', () => {
        const { container } = render(<Other />);
        expect(container).toMatchSnapshot();
    });
});
