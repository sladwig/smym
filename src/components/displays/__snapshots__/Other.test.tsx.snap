// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Other renders 1`] = `<div />`;
